tests/unit/test___init__.py

```diff
@@ -4,6 +4,17 @@
 def test_get_demo():
     demo = get_demo()
 
-    assert list(demo.columns) == ['0_int', '1_float', '2_str', '3_datetime']
-    assert len(demo) == 10
-    assert list(demo.isna().sum(axis=0)) == [2, 2, 2, 2]
+    assert list(demo.columns) == [
+        'last_login', 'email_optin', 'credit_card', 'age', 'dollars_spent'
+    ]
+    assert len(demo) == 5
+    assert list(demo.isna().sum(axis=0)) == [1, 1, 1, 0, 1]
+
+
+def test_get_demo_many_rows():
```

Comment:
Could we also add a test to make that the same rows are returned every time? -- ie validate that fixing the seed is working for everything we generate